platform/platform-migratable/src/main/java/org/codice/ddf/platform/migratable/impl/PlatformMigratable.java

```diff
@@ -47,28 +52,38 @@
 
   private static final Path BIN_DIR = Paths.get("bin");
 
+  private static final List<String> SYSTEM_PROPERTIES_TO_PRESERVE =
+      ImmutableList.of("solr.password");
+
+  private static final Path SYSTEM_PROPERTIES_PATH = Paths.get("etc", "custom.system.properties");
+
   private static final List<Path> REQUIRED_SYSTEM_PATHS =
       ImmutableList.of( //
           Paths.get("etc", "ws-security"),
           Paths.get("etc", "system.properties"),
-          Paths.get("etc", "custom.system.properties"),
+          SYSTEM_PROPERTIES_PATH,
           Paths.get("etc", "startup.properties"),
           Paths.get("etc", "custom.properties"),
           Paths.get("etc", "config.properties"));
 
-  private static final List<Path> OPTIONAL_SYSTEM_PATHS =
+  private static final List<Path> UPGRADEABLE_SYSTEM_PATHS =
       ImmutableList.of( //
-          Paths.get("etc", "users.properties"),
-          Paths.get("etc", "users.attributes"),
-          Paths.get("etc", "pdp", "ddf-metacard-attribute-ruleset.cfg"),
-          Paths.get("etc", "pdp", "ddf-user-attribute-ruleset.cfg"),
-          Paths.get("etc", "org.codice.ddf.admin.applicationlist.properties"),
-          Paths.get("etc", "fipsToIso.properties"),
-          Paths.get("etc", "log4j2.xml"),
-          Paths.get("etc", "certs", "meta"),
-          Paths.get("etc", "certs", "1"),
-          Paths.get("bin", "karaf"),
-          Paths.get("bin", "karaf.bat"));
+          Paths.get("etc", "users.properties"), Paths.get("etc", "users.attributes"));
```

Comment:
Should these files be included in the documentation and the respective DDF issue for files that are imported during a different version upgrade?

Reply:
Yes they should be. I missed them for some reason. Good catch! 👍